site/content/blogs/release-v132.en.md

```diff
@@ -35,8 +35,9 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
 ## 
 
 
-## 
+## `copilot run local --watch`
 
+With the `--watch` flag, Copilot will watch your workspace for file changes and rebuild your containers for you so you can develop continuously.
```

Comment:
We should either add the anchor link above, or maybe since this isn't very long, add these details to the blurb above and then we won't need a detailed section.

Reply:
Mention how it's great with `--proxy`! 🤣 I know the features is super self-explanatory but it's so good I hope people know how much we like it 🥺

Reply:
Good point 😃 I'll revert this last change and make that note!